hhnk_threedi_tools/core/climate_scenarios/klimaatsommen_prep.py

```diff
@@ -25,7 +25,7 @@ def __init__(
         if type(cfg_file) == str:
```

Comment:
- ongebruikte imports verwijderen
- `self = KlimaatsommenPrep(...` vervangen voor `ksp = KlimaatsommenPrep(...` (of iets anders)

Reply:
self staat alleen onder __name__==main, handig voor testen